recipe/meta.yaml

```
@@ -29,10 +29,10 @@ requirements:
   host:
     - python
     - pip
-    - setuptools
+    - setuptools >=42
     - wheel
     - scikit-build
```

I think it should be a range on the 0.17.0 version?
```suggestion
    - scikit-build >=0.17.0,<0.18.0
```

---

True, though I did do this on purpose.  Please see my note up above.  I actually don't think it is strictly required.